ddapm_test_agent/agent.py

```diff
@@ -142,6 +141,8 @@ async def _prepare_and_send_request(data, request, headers):
     agent_url = request.app["agent_url"]
     full_agent_url = agent_url + request.path
     log.info("Forwarding request to agent at %r", full_agent_url)
+    if "DD_API_KEY" not in headers:
```

Comment:
why was this added? it's not used anywhere?

Reply:
removed, cant remember tbh